src/java_bytecode/java_bytecode_internal_additions.cpp

```diff
@@ -41,4 +43,16 @@ void java_internal_additions(symbol_table_baset &dest)
     symbol.is_thread_local=true;
     dest.add(symbol);
   }
+
+  {
+    auxiliary_symbolt symbol;
+    symbol.base_name = "@inflight_exception";
```

Comment:
Nit: consider adding an extra define for just this suffix that is used in `INFLIGHT_EXCEPTION_VARIABLE_NAME` to ensure they match up.